packages/protocol/contracts/interfaces/IAZTEC.sol

```diff
@@ -2,12 +2,20 @@ pragma solidity >=0.5.0 <0.6.0;
 
 contract IAZTEC {
     enum ProofCategory {
-        BALANCED,
-        MINT,
-        BURN,
-        UTILITY
+        BALANCED, // 0
+        MINT,     // 1
+        BURN,     // 2
+        UTILITY   // 3
     }
-
+    // proofEpoch = 1 | proofCategory = 0 | proofId = 1
     // 1 * 256**(2) + 0 * 256**(1) ++ 1 * 256**(0)
     uint24 public constant JOIN_SPLIT_PROOF = 65537;
```

Comment:
might help to represent these ints in their hex form: 0x010001 (0x010101 after the category changes in master branch)